### PR TITLE
ADD OPEN DEX

### PR DESCRIPTION
**Pull request by opendex-labs**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

##### Twitter Link:
N/A

##### List of audit links if any:
N/A

##### Website Link:
https://open-dex.com/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders): 
https://open-dex.com/favicon.png

##### Current TVL:
$307k

##### Chain:
Nova Network

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
N/A

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
N/A

##### Short Description (to be shown on DefiLlama):
The first truly decentralized and open exchange with open token listing.

##### Token address and ticker if any:
0xe36118ccfa51e4caf750dfaeaf5c6cf250759acb
ODX

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
Fusion Boilerplate

##### methodology (what is being counted as tvl, how is tvl being calculated):
Total liquidity deposited in the pools.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/open-dex exports TVL: 
        
 
 
    nova-staking              0
    nova                      261.64 k
    staking                   0
    
    total                    261.64 k

File: projects/open-dex/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const { getUniTVL, sumUnknownTokens } = require("../helper/unknownTokens");
+const sntNova = "0x657a66332a65b535da6c5d67b8cd1d410c161a08";
+const odxToken = "0xe36118ccfa51e4caf750dfaeaf5c6cf250759acb";
+const qsrToken = "0x356c044B99e9378C1B28A1cAb2F95Cd65E877F33";
+const nUSD = "0x1F5396f254EE25377A5C1b9c6BfF5f44e9294fFF"
+const dexFactory = "0x9550b0c83AD5a58898cD4267987Af67e7E52bF55";
```
Comment (g1nt0ki) [inline projects/open-dex/index.js:6]: this is fusion dex factory address, what is open dex factory address?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/open-dex/index.js (hunk 1 above)
Comment (opendex-labs) [reply projects/open-dex/index.js:6]: it routes it via aggregation, uses fusion factory

Comment (g1nt0ki) [reply projects/open-dex/index.js:6]: sorry, not sure if you are realizing it, current PR is counting all of fusion tvl as yours

if your UI supports creating pools using their factory, the new tvl is still theirs.

Comment (opendex-labs) [reply projects/open-dex/index.js:6]: yes, its an aggregator, i thought you would add it as you do with yield optimizers or staking protocols, with the double-counting toggle and warning, do i need to incorporate that into my adaptor?

Comment (g1nt0ki) [reply projects/open-dex/index.js:6]: sorry, yield optimizer have their own contracts, we dont count count aggregator unless they have their own tvl

**Pull request closed without merging**